ENIGMAsystem/SHELL/Platforms/Win32/WINDOWSwindow.cpp

```diff
@@ -481,6 +485,13 @@ void window_mouse_set(int x, int y)
 
 }
 
+static void keyboard_set_direct(int key, bool down) {
+  UINT scancode = MapVirtualKey(key, MAPVK_VK_TO_VSC_EX);
+  DWORD flags = ((scancode >> 8) == 0xE0) ? KEYEVENTF_EXTENDEDKEY : 0;
```

Comment:
I don't know what E0 is, either.

Reply:
You can thank the IBM Model M for `0xE0` Josh. When they started adding more keys than the standard IBM w/e layout, they referred to those keys as "extended" and the high byte is `0xE0` prefixed to the scan code.
https://en.wikipedia.org/wiki/Scancode#PC_compatibles

There's also `0xE1` too, but because the documentation for `KEYEVENTF_EXTENDEDKEY` specifically only mentions `0xE0` I won't go overboard.